helm/goldilocks-app/values.yaml

```diff
@@ -1,5 +1,120 @@
-# Default values for goldilocks-app.
+# uninstallVPA -- Enabling this flag will remove a vpa installation that was previously managed with this chart. It is considered deprecated and will be removed in a later release.
+uninstallVPA: false
 
-goldilocks:
-  image:
-    repository: quay.io/giantswarm/goldilocks
+vpa:
+  # vpa.enabled -- If true, the vpa will be installed as a sub-chart
+  enabled: false
+
+image:
+  # image.repository -- Repository for the goldilocks image
+  repository: quay.io/giantswarm/goldilocks
```

Comment:
We have a pattern for images -> https://github.com/giantswarm/installations/blob/master/default/appcatalog/default-appcatalog-values.yaml#L21 

This ensures that China clusters use the right repo